modules/aleph/lib/AlephClient/AlephClient.class.php

```diff
@@ -181,6 +176,31 @@ public function authenticate($bor_id, $verification, $library = 'ICE53', $sub_li
     return $return;
   }
 
+  /**
+   * Get dom elements by tag name.
+   *
+   * @param array $tags
+   *    The tags for the values you want to have returned.
+   *
+   * @return array
+   *    Array with the tag as key and the value as value or an empty array.
+   */
+  public function get($tags) {
+    // Array to store the returned key/value pairs.
+    $tag_values = array();
+
+    // Loop through each provided tag.
+    foreach ($tags as $tag) {
+      // Extract each value from all the tags.
+      if ($results = $this->dom->getElementsByTagName($tag)) {
+        foreach ($results as $result) {
+          $tag_values[$tag] = $result->nodeValue;
```

Comment:
Seems like you don't really handle the case where you have multiple matches. At the very least you should probably add to the description that the last match will be returned in case of multiple matches.